data/change-streams/unified/change-streams.yml

```diff
@@ -1,6 +1,6 @@
 description: "change-streams"
 
-schemaVersion: "1.7"
+schemaVersion: "1.4"
```

Comment:
Our copy of `change-streams.yml` is not fully up-to-date with the [version](https://github.com/mongodb/specifications/blob/master/source/change-streams/tests/unified/change-streams.yml) in `specifications`. In particular, we do not have instances of the `ignoreExtraEvents` flag that is part of the `1.7` unified test format schema version. Rather than bump our supported schema version to 1.7, I'll keep these tests on 1.4 (which is required for the `serverless: forbid` run-on requirement.